templates/home/reports/pre-a11y-1.json

```
@@ -0,0 +1,444 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/templates/home/home-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Assessing the main content of the GC Home Page template",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": "#wb-main"
+    }
+  },
+
+  "earl:assertedBy": {
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Jonas Graham (Github: @jonasg-gc)",
+      "foaf:homepage": "https://github.com/jonasg-gc",
+      "@type": ["earl:Assertor", "foaf:Person"]
+    }
+  },
+
+  "dct:date": "2023-07-14",
+  "dct:description": "Analyzing and exploring the subject and produce a pre-evaluation of all WCAG 2.1 SC at level AA.",
+  "acr:involvesExpertise": [
+    "wbv:acr/topics#aria",
+    "Javascript"
+  ],
```

was there an special expertise of ARIA and Javascript required to assess this home page? I know I added those 2 as an example so probably not. 

If that is the case, just leave the array empty like
```suggestion
  "acr:involvesExpertise": [  ],
```